src/APIBasedAuth.ts

```diff
@@ -2,14 +2,14 @@ import axios, { AxiosInstance, AxiosResponse } from 'axios';
 import { DEFAULT_BASE_URL } from './utils/helper';
 
 export const API_AUTH_STORAGE_KEY = 'lo-app-tools-api-auth';
-const SESSION_KEYS = ['session', 'username'];
+const SESSION_KEYS = ['session', 'username'] as const;
```

Comment:
this fixes some types within this code that were using this and returning `any` types instead of proper types